crates/cli/src/commands/install.rs

```diff
@@ -88,6 +97,61 @@ mod tests {
     use pacquet_package_json::{DependencyGroup, PackageJson};
     use tempfile::tempdir;
 
+    #[test]
+    fn install_args_to_dependency_groups() {
+        use DependencyGroup::{Default, Dev, Optional};
+        let create_list =
+            |args: InstallCommandArgs| args.get_dependency_groups().collect::<Vec<_>>();
+
+        // no flags -> prod + dev + optional
+        assert_eq!(
+            create_list(InstallCommandArgs { prod: false, dev: false, no_optional: false }),
+            [Default, Dev, Optional],
+        );
+
+        // --prod -> prod + optional
+        assert_eq!(
+            create_list(InstallCommandArgs { prod: true, dev: false, no_optional: false }),
+            [Default, Optional],
+        );
+
+        // --dev -> dev + optional
+        assert_eq!(
+            create_list(InstallCommandArgs { prod: false, dev: true, no_optional: false }),
+            [Dev, Optional],
+        );
+
+        // --no-optional -> prod + dev
+        assert_eq!(
+            create_list(InstallCommandArgs { prod: false, dev: false, no_optional: true }),
+            [Default, Dev],
+        );
+
+        // --prod --no-optional -> prod
+        assert_eq!(
+            create_list(InstallCommandArgs { prod: true, dev: false, no_optional: true }),
+            [Default],
+        );
+
+        // --dev --no-optional -> dev
+        assert_eq!(
+            create_list(InstallCommandArgs { prod: false, dev: true, no_optional: true }),
+            [Dev],
+        );
+
+        // --prod --dev -> prod + dev
+        assert_eq!(
+            create_list(InstallCommandArgs { prod: true, dev: true, no_optional: false }),
+            [Default, Dev],
+        );
+
+        // --prod --dev --no-optional -> prod + dev
+        assert_eq!(
+            create_list(InstallCommandArgs { prod: true, dev: true, no_optional: false }),
+            [Default, Dev],
+        );
```

Comment:
@zkochan @anonrig Do you agree on this implementation, or do you want to change?

Reply:
almost, added one comment.

Reply:
I have followed @zkochan's suggestion.